xc/device.ml

```diff
@@ -1244,8 +1244,10 @@ let clean_shutdown (task: Xenops_task.t) ~xs (x: device) =
 			try (Xenctrl.domain_getinfo xc x.frontend.domid).Xenctrl.hvm_guest
 			with _ -> false
 			in
-		try release ~xc ~xs ~hvm devs x.frontend.domid x.frontend.devid
-		with _ -> ());
+		if hvm then 
```

Comment:
I might be wrong but I didn't see this PCI.clean_shutdown/hard_shutdown used anywhere in other parts of xenopsd.  Should we just try to remove them?